### ISSUE TITLE
3.2K RL training subset partitioning in the paper

### ISSUE THREAD

**Issue by empty7777** [NOVAglow646/Monet#15]
Hello, could you please explain how the 3.2K data subset used in the RL training phase was partitioned, and how the validation set was partitioned? I noticed that the Readme describes the entire Thyme-RL dataset as being downloaded directly, without explaining how the 3.2K data subset and validation set were partitioned as described in the paper. Thank you.

**Comment by ZHUXUHAN** [comment]
> Hello, could you please explain how the 3.2K data subset used in the RL training phase was partitioned, and how the validation set was partitioned? I noticed that the Readme describes the entire Thyme-RL dataset as being downloaded directly, without explaining how the 3.2K data subset and validation set were partitioned as described in the paper. Thank you.

I have the same problem.

**Comment by NOVAglow646** [comment]
We randomly select the training examples from the whole training set of Thyme-RL with a fixed seed (see line47˜51 of `RL/verl/trainer/data_loader.py`). We run the RL training for 50 steps with a batch size of 64, so there are 3.2K training samples in total.
